app-games/src/main/java/com/aptoide/android/aptoidegames/gamegenie/presentation/GameGenieViewModel.kt

```diff
@@ -35,6 +38,12 @@ class GameGenieViewModel @Inject constructor(
   private val _firstLoad = MutableStateFlow(true)
   val firstLoad: Flow<Boolean> = _firstLoad
 
+  private val _installedGames = MutableStateFlow<List<MyGamesAppImage>>(emptyList())
+  val installedGames = _installedGames.asStateFlow()
+
+  private val _selectedGame = MutableStateFlow<MyGamesAppImage?>(null)
```

Comment:
like stated on the view, could this be part of the ui state? containing all the info necessary for the view, same thing as the installed games above?
the problem is that this would require for the things to be emitted around the same time, we can discuss this after